apps/frontend/src/pages/projects/projects.tsx

```diff
@@ -1,118 +1,45 @@
-import {
-	Header,
-	Loader,
-	Modal,
-	Select,
-	Sidebar,
-	Table,
-} from "~/libs/components/components.js";
-import {
-	mockColumns as mockTableColumns,
-	mockData as mockTableData,
-	type Person,
-} from "~/libs/components/table/mock-data.js";
-import { SIDEBAR_ITEMS } from "~/libs/constants/constants.js";
-import { AppRoute } from "~/libs/enums/enums.js";
+import { Loader, PageLayout } from "~/libs/components/components.js";
+import { DataStatus } from "~/libs/enums/enums.js";
 import {
 	useAppDispatch,
-	useAppForm,
 	useAppSelector,
 	useEffect,
-	useLocation,
-	useModal,
 } from "~/libs/hooks/hooks.js";
-import { type SelectOption } from "~/libs/types/types.js";
-import { actions as userActions } from "~/modules/users/users.js";
+import { actions as projectActions } from "~/modules/projects/projects.js";
+
+import { ProjectCard } from "./components/components.js";
+import styles from "./styles.module.css";
 
 const Projects = (): JSX.Element => {
 	const dispatch = useAppDispatch();
-	const { pathname } = useLocation();
-	const dataStatus = useAppSelector(({ users }) => users.dataStatus);
-	const users = useAppSelector(({ users }) => users.users);
-	const { isModalOpened, onModalClose, onModalOpen } = useModal();
 
-	const isRoot = pathname === AppRoute.ROOT;
-	const isLoading = dataStatus === "pending";
-
-	const { control } = useAppForm({
-		defaultValues: {
-			options: [],
-		},
-	});
+	const { dataStatus, projects } = useAppSelector(({ projects }) => ({
+		dataStatus: projects.dataStatus,
+		projects: projects.projects,
+	}));
```

Comment:
If I'm not wrong this can be simplified to 
```suggestion
	const { dataStatus, projects } = useAppSelector(({ projects }) => projects);
```
with no trade-off thanks to destructuring syntax. I mean, that if there will be more fields in `projects` slice, and some of them (except destructured ones) change, this component won't rerender, as it is in your implementation (if I'm not wrong)

Reply:
agree

Reply:
simplified, thanks